scripts/windows/cleanup.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+echo "Stopping Firebird Server..."
+sc query FirebirdServerTestInstance >nul 2>&1
+if errorlevel 1 (
+    echo "FirebirdServerTestInstance Service does not exist. Nothing to do."
+) else (
+	sc stop FirebirdServerTestInstance
+	sc delete FirebirdServerTestInstance
+	echo "Stopped FirebirdServerTestInstance"
+)
+
+echo "Stopping SNMPTrap..."
+sc query SNMPTrap >nul 2>&1
+if errorlevel 1 (
+    echo "SNMPTrap Service does not exist. Nothing to do."
+) else (
+	sc stop SNMPTrap
+	sc delete SNMPTrap
+	echo "Stopped SNMPTrap"
+)
+
+if EXIST C:\tests_tmp rmdir /s /q C:\tests_tmp
+if EXIST %PHP_BUILD_DIR%\test_file_cache rmdir /s /q %PHP_BUILD_DIR%\test_file_cache
```

Comment:
I run a separate version of this locally. This is like the <10th iteration, but I want to get folks thinking on it. The tests crowd people's machines with lots of (I think frankly trash); but I can also see to make this fully flexible, folks really should have the choice to just turn off a service.

How then to ensure the services names are known globally rather than shared by convention like this?